lib/db/mem.js

```diff
@@ -794,6 +795,58 @@ module.exports = function (log, error) {
     return P.resolve({})
   }
 
+  // VERIFICATION REMINDERS
+
+  Memory.prototype.createVerificationReminder = function (body) {
+    if (! body || ! body.uid || ! body.type) {
+      throw new Error('"uid", "type" are required')
+    }
+
+    var reminderData = {
+      uid: body.uid,
+      type: body.type,
+      createdAt: Date.now()
+    }
+    reminders[reminderData.uid.toString('hex') + reminderData.type] = reminderData
+
+    return P.resolve({})
+  }
+
+  Memory.prototype.fetchReminders = function (body, query) {
+    if (! query || ! query.reminderTime || ! query.type || ! query.limit) {
+      throw error.wrap(new Error('fetchReminders - reminderTime, limit or type missing'))
```

Comment:
Noticed that some places use `error.wrap` and some places just throw, is this intended?